.github/workflows/bonsai.yml

```diff
@@ -40,6 +40,15 @@ jobs:
       - name: Install Foundry
         uses: risc0/foundry-toolchain@2fe7e70b520f62368a0e3c464f997df07ede420f
 
+      - name: Install cargo-install
+        uses: risc0/cargo-install@v1
+        with:
+          crate: cargo-binstall
+          version: "1.4"
+
+      - name: Cargo binstall cargo-risczero
+        run: cargo binstall -y cargo-risczero
```

Comment:
```suggestion
        run: cargo binstall -y cargo-risczero@0.19.0
```

Reply:
Actually, I think we should use the current version from source instead of the latest published one so that we can detect breaking changes in CI.